run-health-checks.sh

```diff
@@ -45,10 +45,10 @@ fi
 
 #add accelerated network if applicable
 NHC_DIR=$(dirname "${BASH_SOURCE[0]}")
-acc_net=$(ibstatus mlx5_an0)
 acc_file="$NHC_DIR/conf/$conf_name.conf"
+acc_net=$(ibstatus mlx5_an0)
 if [ $? -eq 0 ] && ! grep -q 'check_hw_ib 40 mlx5_an0:1' "$acc_file"; then
-    echo -e "\n\n### Accelerate network check\n * || check_hw_ib 40 mlx5_an0:1\n * || check_hw_eth eth1">> $acc_file
+	echo -e "\n\n### Accelerate network check\n * || check_hw_ib 40 mlx5_an0:1\n * || check_hw_eth eth1">> $acc_file
```

Comment:
hmm, what did we change here? or we just unify the indentation here?

Reply:
I think this is a formatting difference from using 2 different editors. let me tr to change it back

Reply:
addressed